content/hi/_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+---
+title: परिभाषा टेम्पलेट
+status: Feedback Appreciated
+category: concept
+---
+
+## यह क्या है
+अवधारणा का एक त्वरित सारांश।
```

Comment:
`संक्षेप` can be used in place of `त्वरित` because it gives a better reference to short summary/quick summary. The literal translation of quick seems wrong.

Reply:
त्वरित सारांश is a commonly used phrase in Hindi too just like quick summary is in English. I have seen HIndi newspapers using the term many times. I'll try to link some examples from the internet if I find.